serial_views.py

```diff
@@ -0,0 +1,153 @@
+# serialize data for all models
+from django.http import HttpResponse
+
+
+import WAM_APP_FRED.oep_models as oep_modles
+# from WAM_APP_FRED.db_sqla import *
+from shapely.wkb import loads as loadswkb
+from geojson import Feature, FeatureCollection, dumps
+from geoalchemy2 import functions
+import sqlahelper as sah
+from datetime import datetime
+
+# def serializer():
+#     """
```

Comment:
This is a simple integration of my serializer should be easy to understand . Will delete this soon.